feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/task.c

```diff
@@ -57,19 +57,24 @@ void task_status(struct task *task, int status, char *result)
 	switch (status) {
 	case TASK_WAITING:
 		SCHEMA_SET_STR(state.state, "waiting");
+		strcpy(task->state,"waiting");
```

Comment:
Why use a string here instead of storing the value of status. eg TASK_WAITING, TASK_PENDING.  You'd save effort on the strcmp() at line 184 as well.

Reply:
got it